MAINTAINERS.md

```diff
@@ -205,36 +290,45 @@ to pass.
 
 - [ ] Push release labels as tags
 
-  Perform the following for each \<TAG> that we will use to label this release.
-
+  Perform the following for each [_**release label**_](#assign-release-parameters):
   ```sh
-  git tag <TAG>
-  git push origin <TAG>
+  git tag <release-label>
+  git push origin <release-label>
   ```
 
-- [ ] Confirm that a Docker image for SDK version $SDKVER has been published to the
+- [ ] Confirm that a Docker image for SDK version [$SDKVER](#user-content-generate-sdk-version)
+  has been published to the
   [agoric-sdk Container registry](https://github.com/Agoric/agoric-sdk/pkgs/container/agoric-sdk).
 
   Note that this is triggered by pushing the `@agoric/sdk@$n` tag in the
-  "Publish the released package tags" step and may take a while.
+  ["Publish the released package tags" step](#user-content-publish-tags) and may take a while.
   You can observe workflow initiation and progress at
   [Build release Docker Images](https://github.com/Agoric/agoric-sdk/actions/workflows/docker.yml).
 
 - [ ] Create a GitHub release
 
   Follow the [GitHub
   instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
-  ) and use previous releases as a template. This uses the
-  validator-oriented release description.
+  ) and use previous releases as a template. The release description should include the
+  [_**validator oriented release description**_](#describe-the-release).
 
 ### Cleanup
 
-- [ ] Review recent changes in the base branch for anything that
-  should be merged into its ancestors, all the way up to master. This
+- [ ] Update the [_**base branch**_](#assign-release-parameters) **a3p-integration/proposals/*/package.json**
+  `agoricProposal` `sdkImageTag` string contents to match the number from the
+  ["Generate new SDK version" step](#user-content-generate-sdk-version).
```

Comment:
It's unclear what to do with this edit, especially given the subsequent request to delete the checkout.

I think this section is better placed as a sub-bullet of copying over the test proposal to agoric-3-proposal, as one of the modifications to be made.

Other modifications:
- set the `releaseNotes` to the expected URL
- generate the expected `upgradeInfo` from the script

Reply:
Done.